contrib/automation_tests/orbit_load_presets.py

```diff
@@ -0,0 +1,160 @@
+"""Apply two presets in Orbit using pywinauto.
```

Comment:
In python, I prefer single-line comments, but if you want to use multi-line strings, you should add a line break after the """.

Reply:
Just looked up python's doc-strings again. And you are right about the format. So forget what I said.